### PR TITLE
Shows legcuffs in examine and lets you see what type of cuffs and bola a mob is restrained with

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Makes legcuffs show up on examine, makes it show what type of hand/legcuff it is. Removes the icon in the examine, it looked bad anyways.

image

## Why It's Good For The Game

Being able to tell if someone's legcuffed from examine is neat. Being able to see what they're restrained with is also neat.

## Changelog
:cl:
add: Legcuffs now show up in examine
add: You can now see what type of restraint the mob is restrained with
/:cl:

### CODE REVIEW

**Comment by KubeRoot** [comment]
Why is the legcuff icon not showing up in examine on the screenshot?

**Comment by BrumalLake** [comment]
animated state most likely

**Comment by KubeRoot** [comment]
Should it be locking the icon to a single frame then?

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -373,3 +375,4 @@
 			dat += "[new_text]\n" //dat.Join("\n") doesn't work here, for some reason
 	if(dat.len)
 		return dat.Join()
+a
```
Comment (KubeRoot) [inline code/modules/mob/living/carbon/human/examine.dm]: I'm in awe at BYOND at the fact that this is compiling

Comment (KubeRoot) [reply code/modules/mob/living/carbon/human/examine.dm]:
```suggestion

```

**Comment by Kapu1178** [comment]
honestly the handcuff icon looks fucking bad, remove the icons of both imo.

File: code/modules/mob/living/carbon/human/examine.dm (hunk 1 above)
Comment (ivanmixo) [reply code/modules/mob/living/carbon/human/examine.dm]: Lmao what the fuck

Comment (francinum) [reply code/modules/mob/living/carbon/human/examine.dm]: It compiles because it's defining a new atom of type `a`.

**Pull request merged**